Drink_Book_PR/DrinkBook/Club.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DrinkBook
+{
+    public abstract class Club
+    {
+        private string name;
+        private int priceOfWhiskey;
+        private int priceOfVodka;
+        private int capacity;
+
+        private string musicPlayed;
+
+        List<User> listOfUsers = new List<User>();
+
+        public string MusicPlayed
+        {
+            get
+            {
+                return musicPlayed;
+            }
+            set
+            {
+                musicPlayed = value;
+            }
+        }
+
+        public string Name
+        {
+            get
+            {
+                return name;
+            }
+            set
+            {
+                name = value;
+            }
+        }
+
+        public int PriceOfWhiskey
```

Comment:
Suggestions
1.  You can use the shorthand syntax for get /set: 
```suggestion
        public int PriceOfWhiskey
        {
             get => this.priceOfWhiskey;
             set => this.priceOfWhiskey = value;
        }
```

2.  You can make use of the Auto-Implemented properties. Ref: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties

Reply:
Also you can set negative values for prices. You should add validation